build.rs

```diff
@@ -1,7 +1,21 @@
 use anyhow::Result;
-use std::env;
+use std::{env, path::Path};
 
 fn build() -> Result<cc::Build> {
+    if !ci_info::is_ci() {
+        // Check for sec scan
+        let prepared_path = Path::new(".prepared_llt_secrets");
+        #[allow(clippy::panic)]
+        if !prepared_path.is_file() {
+            match env::var("BYPASS_LLT_SCAN") {
+                Ok(_) => println!("cargo:warning=BYPASS_LLT_SCAN IS SET, COMMIT CAREFULLY!!"),
+                Err(_) => {
+                    panic!("Hooks not found, either run checkout scripts or run with BYPASS_LLT_SCAN=1");
```

Comment:
nit: this code only requires that the `BYPASS_LLT_SCAN` exists, not that it has the value `1`. Either change the code or adjust the message.

Reply:
you're right, message changed.